src/netex_france/stops.rs

```diff
@@ -350,14 +350,21 @@ impl<'a> StopExporter<'a> {
         None
     }
 
-    fn generate_quay_accessibility(&self, equipment_id: Option<&'a String>) -> Option<Element> {
+    fn generate_quay_accessibility(
+        &self,
+        stop_point_id: &'a str,
+        equipment_id: Option<&'a String>,
+    ) -> Option<Element> {
```

Comment:
Send the reference to whole `StopPoint` instead since you need 2 fields out of it...
```suggestion
    fn generate_quay_accessibility(
        &self,
        stop_point: &'a StopPoint,
    ) -> Option<Element> {
```

Reply:
Ah, well seen! I missed the idea :+1:

Reply:
Fix in d95424d438267811d3f4b3185f71f22e924d90f0